clerk.cc

```diff
@@ -139,17 +161,24 @@ int main(int argc, char** argv) {
     sender.reset(new clerk::PacketSender(fd, &factory));
   }
 
-  clerk::TestimonyProcessor processor(FLAGS_testimony, &factory);
+  clerk::Processor* processor;
+  if (FLAGS_testimony != "") {
+     processor = new clerk::TestimonyProcessor(FLAGS_testimony, &factory);
+  } else {
+     processor = new clerk::FlowGenerator(FLAGS_upload_every_secs,
+           FLAGS_flow_timeout_secs, FLAGS_num_hosts, FLAGS_num_flows_per_host,
+           &factory);
+  }
   double last_upload_secs = GetCurrentTimeSeconds();
```

Comment:
Since processor is a pointer type now. Do you want to delete it at some point for downstream destructotors to be called?

Reply:
Good catch. Fixed.